src/main/resources/db/changesets/20200512_reference_unique_index.yaml

```diff
@@ -0,0 +1,17 @@
+databaseChangeLog:
```

Comment:
figured since there's a dev instance it's time to start adding new changesets

Reply:
Yeah, I think that's the right call